plugin-flex-ts-template-v2/src/feature-library/keyboard-shortcuts/custom-components/KeyboardShortcuts/Tabs/CustomKeyboardShortcutsView.tsx

```diff
@@ -0,0 +1,173 @@
+import DeleteButton from '../DeleteButton';
+import { ShortcutsObject } from '../../../types/types';
+import { getCustomShortcuts } from '../../../utils/KeyboardShortcutsUtil';
+
+interface CustomKeyboardShortcutsViewProps {
+  reset: boolean;
+  disableShortcuts: boolean;
+  isThrottleEnabled: boolean;
+  isDeleteShortcutsEnabled: boolean;
+  toasterDeleteNotification: (actionName: string) => void;
+  toasterSuccessNotification: (actionName: string, oldShortcut: string, newShortcut: string) => void;
+}
+
+const CustomKeyboardShortcutsView = ({
+  reset,
+  disableShortcuts,
+  isThrottleEnabled,
+  isDeleteShortcutsEnabled,
+  toasterDeleteNotification,
+  toasterSuccessNotification,
+}: CustomKeyboardShortcutsViewProps) => {
+  const [customShortcuts, setCustomShortcuts] = useState<ShortcutsObject[]>([]);
+  const [shortcutsDeleted, setShortcutsDeleted] = useState<boolean>(false);
+  const [isEditModalOpen, setIsEditModalOpen] = useState<boolean>(false);
+  const [selectedShortcutKey, setSelectedShortcutKey] = useState<string>('');
+  const [selectedActionName, setSelectedActionName] = useState<string>('');
+  const [selectedThrottle, setSelectedThrottle] = useState<number | undefined>(0);
+
+  const openModalHandler = (shortcut: string, actionName: string, throttle: number | undefined): void => {
+    setIsEditModalOpen(true);
+    setSelectedThrottle(throttle);
+    setSelectedShortcutKey(shortcut);
+    setSelectedActionName(actionName);
+  };
+
+  const closeModalHandler = (): void => {
+    setIsEditModalOpen(false);
+  };
+
+  useEffect(() => {
+    setCustomShortcuts(getCustomShortcuts());
+    setShortcutsDeleted(false);
+  }, [reset]);
+
+  useEffect(() => {
+    if (Object.keys(getCustomShortcuts()).length === 0) {
+      setShortcutsDeleted(true);
+    }
+  }, [customShortcuts]);
+
+  return (
+    <>
+      {/* Check if shortcuts have been disabled or deleted  */}
+      {disableShortcuts || shortcutsDeleted ? (
+        <Card>
+          <Heading as="h5" variant="heading50">
+            <Stack orientation="horizontal" spacing="space20">
+              <WarningIcon decorative={false} title="Description of icon" />
+              Re-enable Keyboard Shortcuts
+            </Stack>
+          </Heading>
+          <Paragraph>
+            There are no configured custom keyboard shortcuts. Please reset your keyboard settings to enable custom
+            keyboard shortcuts.
+          </Paragraph>
+        </Card>
+      ) : (
+        <>
+          {/* Present a table if there are shortcuts to present */}
+          <Box overflowY="auto" maxHeight="1000px">
+            <Table>
+              <THead>
+                <Tr>
+                  <Th>
+                    <Stack orientation="horizontal" spacing="space30">
+                      <Tooltip text="Ctrl and Shift are the default modifiers that cannot be changed.">
+                        <Text as="span">
+                          Modifiers <InformationIcon decorative />
```

Comment:
The icon renders on a newline for me, could it be inline with the text?

Reply:
Yes it can! I just fixed it for both views.